uktrade/data-hub-frontend
# Company details change request

## Description of change

This PR will update the company business details edit form for matched companies which after submission will create Zendesk tickets for each updated field.

## Test instructions

1. Go to a matched company
2. Go to the business details edit page
3. Change some field, e.g. "Trading names"
4. Check if a Zendesk ticket was created (check our staging zendesk instance)
